packages/@uppy/dashboard/src/components/FileCard/index.js

```diff
@@ -154,6 +166,7 @@ class FileCard extends Component {
             <button
               className="uppy-u-reset uppy-c-btn uppy-c-btn-primary uppy-Dashboard-FileCard-actionsBtn"
               type="submit"
+              onClick={'form' in HTMLButtonElement.prototype ? undefined : this.handleSave}
```

Comment:
I think this might be a case where a comment with some context will help future contributors.

Reply:
What do you think of https://github.com/transloadit/uppy/pull/3113/commits/bd7ca31433ed7f089ba2ac1ddac28a09646190da? Open to better suggestions if that do not suffice.